DOCS: document P3-D65 primaries and two missed in format

ValZapod opened the pull request:
display-p3 was undocumented and EVERYONE somehow uses dci-p3 with funny DCI whitepoint, that no one should target and no iPhone photo uses! Oogh.

Natural-Harmonia-Gropius commented:
as https://github.com/mpv-player/mpv/issues/9620#issuecomment-1021971768.
`--target-prim` is primary only, doesn't matter what tfc, wp, cct it is.
rather than call it display p3, i'd like to call it dci-p3-d65
https://en.wikipedia.org/wiki/DCI-P3#System_colorimetry

haasn commented on DOCS/man/options.rst:
```diff
@@ -6180,15 +6180,15 @@ them.
         ITU-R BT.709 (HD), IEC 61966-2-4 (sRGB), SMPTE RP177 Annex B
     bt.2020
-        ITU-R BT.2020 (UHD)
+        ITU-R BT.2020 (UHD), that is the usual HDR space
```
This comment seems unnecessary to me - somebody using this option should know what they're doing, no?

ValZapod replied:
No. Everybody is using display-p3, or dci-p3 (though looks like in that case it was real DCI whitepoint, wow). See this for example:

haasn commented on DOCS/man/options.rst:
```diff
@@ -6198,7 +6198,9 @@ them.
-        DCI-P3 (Digital Cinema Colorspace), SMPTE RP431-2
+        DCI-P3 (Digital Cinema Colorspace), SMPTE RP431-2, DCI whitepoint
+    display-p3
+        P3-D65, SMPTE EG432-1, Netflix masters use with PQ, all Apple devices
```
"Netflix masters use with PQ, all Apple devices" seems unnecessary - no other option is documented this way. Just describe it as it is, i.e. "DCI-P3 with D65 whitepoint"

ValZapod replied:
DCI-P3 uses DCI whitepoint, that is the whole point. I documented whitepoints that are not D65.

Natural-Harmonia-Gropius replied:
maybe looks wired, it's dci-p3-dci and dci-p3-d65, dci in prefix meaning "Digital Cinema Initiatives" not dci wp.

ValZapod replied:
No one uses dci-p3-dci? See wikipedia.

Natural-Harmonia-Gropius replied:
nobody argue about **use** dci or not.
i just don't want to see you make two p3 **primary** looks like totally different things in document.

I like haasn's proposal, and move SMPTE EG432-1 to start make styles consistent.

ValZapod replied:
>two p3 primary looks like totally different things in document.

But they are totally different due to chromatic adaptation due to a different white point.
Morover if we are talking about ICC profiles we always chromatically adapt values there to D50, so D65 white point becomes D50, primaries are shifted and in case of DCI white point it is not on plankian locus, so not CIE D series illuminant even. Additional field chad (chromatic adaptation is added and in case of DCI actually required).

Natural-Harmonia-Gropius replied:
Talk is cheap. Show you the code.
https://github.com/ampas/aces-dev/blob/dev/transforms/ctl/lib/ACESlib.Utilities_Color.ctl#L41-L63

ValZapod replied:
Yes, ACES uses D60, which is perfect Series D: 0.32168,  0.33767 and is on plankian locus.

But the DCI is not.

>SMPTE EG432-1 to start make styles consistent

No, H.273 shows 431-2 first.

Natural-Harmonia-Gropius replied:
```
        SMPTE EG432-1, SMPTE RP431-2, P3-DCI (Theater)
    display-p3
        SMPTE EG432-1, SMPTE RP431-2, P3-D65 (Display)
```
My opinion here. If you don't accept it, ignore it, it's your pr.

haasn commented on DOCS/man/vf.rst:
```diff
@@ -247,12 +247,13 @@ Available mpv-only filters are:
         :bt.601-625:   ITU-R BT.601 (SD) 625-line systems (PAL, SECAM)
         :bt.709:       ITU-R BT.709 (HD) (same primaries as sRGB)
-        :bt.2020:      ITU-R BT.2020 (UHD)
+        :bt.2020:      ITU-R BT.2020 (UHD), default for HDR transfers
```
This is not true, mpv does not default the primaries based on the transfer

ValZapod replied:
Yes, that is jot mpv comment though.

haasn commented on DOCS/man/vf.rst:
```diff
@@ -247,12 +247,13 @@ Available mpv-only filters are:
         :prophoto:     ProPhoto RGB (ROMM)
         :cie1931:      CIE 1931 RGB
         :dci-p3:       DCI-P3 (Digital Cinema)
+        :display-p3:   P3-D65 (all modern Apple, use with PQ by Netflix masters)
```
See above, this seems unnecessary and goes against the convention of the rest of the docs

haasn commented:
Commits should be squashed

ValZapod commented:
> rather than call it display p3, i'd like to call it dci-p3-d65

We cannot this is part of public API now. What is done is done. See https://github.com/mpv-player/mpv/pull/5236#issuecomment-351904328

**Anyway, squashed and addressed comments**. I WILL ADD white points to those two places, sorry. BTW, is there a way to give any white point and primaries to it??? Like Little CMS should not care, does it not??

Also WTF is that??

https://github.com/mpv-player/mpv/blob/6265724f3331e3dee8d9ec2b6639def5004a5fa2/video/csputils.c#L942-L946

haasn commented:
Change seems fine to me

ValZapod commented:
@Dudemanguy please merge

ValZapod commented:
@jeeb Please merge.

quietvoid commented:
Any update on this? I'm interested in having the `display-p3` option.

haasn commented:
> Any update on this? I'm interested in having the `display-p3` option.

The *option* already exists. This patch just adds the manpage entry for it.

I will probably merge it eventually. Some day. I think @ValZapod is pushy enough with his comments that he can learn to wait.

quietvoid commented:
Oh okay, I thought it wasn't being passed to VOs. Thanks.

ValZapod commented:
> comments that he can learn to wait.

I cannot see your comment. looks like you forgot you blocked. Hillarious.
